applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -290,26 +336,26 @@ static void work_stream_start(struct k_work *work)
 static void stream_enabled_cb(struct bt_audio_stream *stream)
 {
 	int ret;
-	uint8_t channel_index;
+	uint8_t stream_index;
 
-	ret = stream_index_get(stream, &channel_index);
+	ret = audio_stream_index_get(stream, &stream_index);
 	if (ret) {
 		LOG_ERR("Stream not found");
 	} else {
 		struct worker_data work_data;
 
-		work_data.channel = channel_index;
+		work_data.stream_index = stream_index;
 		work_data.retries = 0;
 
 		ret = k_msgq_put(&kwork_msgq, &work_data, K_NO_WAIT);
 		if (ret) {
 			LOG_ERR("No space in the queue for the bond");
 		}
 
-		if (channel_index == 0)
-			k_work_schedule(&stream_start_work[channel_index], K_MSEC(500));
+		if (stream_index == 0)
```

Comment:
I need some explanation here.

Reply:
We can't start both streams at the same time, since the controller is busy. Added brackets

Reply:
We will handle this as a separate task.